src/main/java/core/basesyntax/IFigureToString.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface IFigureToString {
+    String toString();
```

Comment:
```suggestion
public interface Drawable {
    void draw();
```
![image](https://github.com/mate-academy/jv-oop-advanced/assets/20326858/bc45e7fe-3780-4559-b447-bff45b4579c7)